cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -843,6 +874,12 @@ impl<T: Config> QueuePausedQuery<ParaId> for Pallet<T> {
 	}
 }
 
+#[derive(Copy, Clone)]
+enum XcmEncoding {
```

Comment:
Some comment please what the variants mean.

Reply:
Done